### ISSUE TITLE
Is there any interest in providing a model trained in Brazilian Portuguese?

### ISSUE THREAD

**Issue by Edresson** [mozilla/TTS#160]
Hello, I recently developed a dataset for voice synthesis in Brazilian Portuguese using my own voice, called TTS-Porguese Corpus, the base has approximately 10 hours of talk, is available at: https://github.com/Edresson/TTS-Portuguese-Corpus

I have already successfully trained the DCTTS model in the dataset.

I created a branch in my repository to support the dataset in this repository, see here: https://github.com/Edresson/TTS/tree/TTS-Portuguese

In initial tests the model unfortunately did not converge, at the moment I'm out of time, and machine to train the model.

  Is anyone interested in training/adjusting the hyperparameters of the model to get a TTS model in Portuguese?

**Comment by erogol** [comment]
Unfortunately not at the moment. Maybe later on.

**Comment by Edresson** [comment]
Hello, I trained the model with the TTS -Portuguese Corpus. Provide a demo here: https://colab.research.google.com/drive/1OZ7T5fqR-QPyN5O6HFetkOeHNCKbdGsA

**Comment by erogol** [comment]
would you mind if I link your model on our released models https://github.com/mozilla/TTS/wiki/Released-Models

**Comment by erogol** [comment]
@Edresson have you tried universal wavernn trained here https://github.com/erogol/WaveRNN? It sounds already good to me but that might improve the results with neural vocoder.

**Comment by Edresson** [comment]
Hello,
You can link my models.
I trained two models. The first model was trained only under the TTS-Portuguese corpus ([demo link](https://colab.research.google.com/drive/1ILnD2j6HJUruqI033suPuBdRY68hKbKK)  ). The second model was trained using transfer learning  (with the model iter-120K | LJSpeech | commit: bf7590) [demo link]( https://colab.research.google.com/drive/1Cbbuz4ggV_sQBs17RXv8ccORcNUsLIne):

About WaveRNN I had been trying to train at TTS-Portuguese Corpus, however, I didn't get good results (I wasn't using transfer learning ). It would be interesting to try its implementation and use  transfer learning.

**Comment by erogol** [comment]
@Edresson which version trained better, the one with transfer learning or the other?

**Comment by Edresson** [comment]
@erogol the version with transfer learning is better.

**Comment by Edresson** [comment]
@erogol I tried to use the WaveRNN universal vocoder. But my pre-trained model uses "mel_fmin" and "mel_fmax: null and" do_trim_silence ": true. While WaveRnn vocoder uses" mel_fmin ": 0.0," mel_fmax ": 8000.0 and" do_trim_silence ": flase. I need to retrain my model ...

Here's the  [TTS + WaveRNN Demo trained with TTS-Portuguese Corpus](https://drive.google.com/file/d/1OHKfmd7uqJ7FyMYkKBFCyT0sXkLm21zv/view?usp=sharing). The audios are understandable but strange. I intend to retrain the model soon and I am updating here.
,

**Comment by Edresson** [comment]
@erol @reuben New pretreined model added to Colab Notebook ( [link](https://drive.google.com/file/d/1OHKfmd7uqJ7FyMYkKBFCyT0sXkLm21zv/view?usp=sharing) ). The notebook generates audio synthesized by the WaveRNN and Griffin-Lim vocoder. WaveRNN audios sound better.
Some issues in stop token loss have been fixed, now the model correctly predicts the pronunciation time of each sentence. I believe it will be the final model for the TTS-Portuguese Corpus. If I have new results I will add it here.

Below the details of the model if you want to add it on the released models page.

[Audio Sample](https://soundcloud.com/user-797601460/sets/tts-mozilla-trained-in-tts-portuguese-corpus-with-wavernn-and-griffinlim-vocoders)
Pretrained Model: [Tacotron2-iter-381000](https://www.dropbox.com/s/91etfwt4tvzjqyz/TTS-checkpoint-phonemizer-wavernn-381000.zip?dl=0) 
Details Model: Tacotron2 + Universal WaveRNN for  Brazilian Portuguese
Commit: [Branch](https://github.com/Edresson/TTS/tree/TTS-Portuguese)

**Comment by pedroalmeida415** [comment]
Hey guys, I was looking for a way to clone a League of Legends character's voice

After some days doing a lot of research, I'm still stuck at where to even begin with

Was wondering if any of you might have any leads for me

Thank you nonetheless!

@Edresson @blue-fish @erogol

**Comment by Cris140** [comment]
Hey @pedroalmeida415, I'm developing a website with cloned voices, there will be some voices from League of Legends as well.
We have a discord in which we can talk about it and help you train the models.
Here's it: https://discord.gg/EUQX5Qfhnm

**Comment by pedroalmeida415** [comment]
Hey @Cris140 sorry for the delay

could you please send me an invite again? would be glad to help with what I can as well

thanks

**Comment by Cris140** [comment]
There you go: https://discord.gg/YCShCG2G

**Comment by juant8001** [comment]
Hey, can you send me a invite ? I really interested on this

**Comment by bebezento** [comment]
@Cris140 Olá Cris140, ainda com o servidor no discord?

**Comment by Cris140** [comment]
@bebezento Com certeza, aqui um novo link: https://discord.com/invite/4npUee2XMk
